### PR TITLE
Improved Russian translation

### PR DESCRIPTION
**Pull request by Inter-Net-Pro**
Added values from file lang\en\lang.php and improved the translation.

### CODE REVIEW

File: lang/ru/lang.php [hunk 1]
```diff
@@ -16,17 +16,27 @@
...
+        'manage_settings' => 'Управление настройками блога',
         'delete_confirm' => 'Вы уверены, что хотите сделать это?',
         'chart_published' => 'Опубликовано',
         'chart_drafts' => 'Черновики',
         'chart_total' => 'Всего',
+        'settings_description' => 'Управление настройками блога',
+        'show_all_posts_label' => 'Показывать все записи для внутренних (бэкенд) пользователей',
+        'show_all_posts_comment' => 'Показывать опубликованные и неопубликованные записи на фронтенде для внутренних (бэкенд) пользователей',
+        'tab_general' => 'Основные'
         'new_category' => 'Новая категория'
```
Comment (LukeTowers) [inline lang/ru/lang.php]: This breaks the language file, please make sure ALL array elements have a trailing comma. Also, please test in the plugin in a running OctoberCMS instance too.

Comment (Inter-Net-Pro) [reply lang/ru/lang.php]: Sorry for the inattention.
Corrected.

File: lang/ru/lang.php [hunk 2]
```diff
@@ -16,17 +16,27 @@
...
         'delete_confirm' => 'Вы уверены, что хотите сделать это?',
         'chart_published' => 'Опубликовано',
         'chart_drafts' => 'Черновики',
         'chart_total' => 'Всего',
-        'new_category' => 'Новая категория'
+        'settings_description' => 'Управление настройками блога',
+        'show_all_posts_label' => 'Показывать все записи для внутренних (бэкенд) пользователей',
+        'show_all_posts_comment' => 'Показывать опубликованные и неопубликованные записи на фронтенде для внутренних (бэкенд) пользователей',
+        'tab_general' => 'Основные'
+
```
Comment (LukeTowers) [inline lang/ru/lang.php]: there's an extra newline here that's unnecessary.

Comment (Inter-Net-Pro) [reply lang/ru/lang.php]: I did as in original file to match lines:
 https://github.com/rainlab/blog-plugin/blob/master/lang/en/lang.php

Delete empty line?

Comment (w20k) [reply lang/ru/lang.php]: Yes, it's incorrect :) No empty lines.

Comment (Inter-Net-Pro) [reply lang/ru/lang.php]: Is done.

**Comment by Inter-Net-Pro** [comment]
Excuse for troubling.
Tell me, do I need to fix something else to accept my request?

**Comment by w20k** [comment]
@Inter-Net-Pro just to wait a little bit, when @LukeTowers will accept review on his end and merge ;)

**Pull request merged**